apps/admin-x-activitypub/src/hooks/useActivityPubQueries.ts

```diff
@@ -166,13 +165,13 @@ export function useFollowersForUser(handle: string) {
     });
 }
 
-export function useAllActivitiesForUser(handle: string) {
+export function useAllActivitiesForUser({handle, includeOwn = false}: {handle: string, includeOwn?: boolean}) {
     const siteUrl = useSiteUrl();
     const api = createActivityPubAPI(handle, siteUrl);
     return useQuery({
-        queryKey: [`activities:${handle}`],
+        queryKey: [`activities:${handle}:includeOwn=${includeOwn.toString()}`],
```

Comment:
Not sure if we have a better scheme for this 🤔

Reply:
Fine for now!